src/features/form/containers/GroupContainer.tsx

```diff
@@ -109,25 +110,29 @@ export function GroupContainer({ id }: IGroupProps): JSX.Element | null {
     }
   }, [dispatch, id, edit?.mode, repeatingGroupIndex, setMultiPageIndex]);
 
-  React.useEffect(() => {
-    if (edit?.openByDefault && repeatingGroupIndex === -1) {
+  useEffect(() => {
+    if (!edit) {
+      return;
+    }
+    if (edit.openByDefault && repeatingGroupIndex === -1) {
       onClickAdd();
     }
-  }, [edit?.openByDefault, onClickAdd, repeatingGroupIndex]);
+  }, [edit, onClickAdd, repeatingGroupIndex]);
```

Comment:
I think we should revert this back to what it was. Since now the dependency array depends on the whole edit object. But the implemented code only depends on `edit.openByDefault`.  Meaning that this `useEffect` runs even when something else than `openByDefault` changes.